webapp/channels/src/components/menu/menu_item.tsx

```diff
@@ -240,7 +240,7 @@ const MenuItemStyled = styled(MuiMenuItem, {
                 '&.Mui-focusVisible .label-elements>:last-child, &.Mui-focusVisible .label-elements>:first-child, &.Mui-focusVisible .label-elements>:only-child': {
                     color: isDestructive && 'var(--button-color)',
                 },
-                '&.Mui-focusVisible .leading-element': {
+                '&.Mui-focusVisible .leading-element, &.Mui-focusVisible .trailing-elements': {
```

Comment:
This fixes the styling of delete button on keyboard down (arrow up/down).

From:
<img width="290" alt="Screenshot 2023-08-10 at 12 37 24 PM" src="https://github.com/mattermost/mattermost/assets/5334504/c346eb7a-0a33-4874-96be-7087337288fc">

To:
<img width="292" alt="Screenshot 2023-08-10 at 12 36 41 PM" src="https://github.com/mattermost/mattermost/assets/5334504/42a181c0-776f-4467-9d94-7c88f44eef8d">